DeviceIdentifier.swift

```diff
@@ -73,14 +73,25 @@ public extension UIDevice {
             }
             #endif
         }
-
+        
         return mapToDevice(identifier: identifier)
     }()
-
+    
 }
 
-struct DeviceInfo {
+struct DeviceIdentifier {
+    static let DEVICE_IDENTIFIER_KEY = "traceDeviceIdentifier"
+    static let RANDOM_STRING_LENGTH = 6
     static func getModel() -> String {
         return UIDevice.modelName
     }
+    static func getID() -> String {
+        if UserDefaults.standard.string(forKey: DEVICE_IDENTIFIER_KEY) == nil {
+            let letters = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+            let randomString = String((0..<RANDOM_STRING_LENGTH).map{ _ in letters.randomElement()! })
+            let deviceIdentifier = "\(randomString) [\(UIDevice.modelName)]"
+            UserDefaults.standard.set(deviceIdentifier, forKey: DEVICE_IDENTIFIER_KEY)
+        }
+        return UserDefaults.standard.string(forKey: DEVICE_IDENTIFIER_KEY)!
+    }
```

Comment:
This only appears to be displayed to the user in the `InfoViewController` and doesn't seem to be sent to the API at any time that I can see?

Reply:
Android sends device Model in the bluetooth payload: https://github.com/AU-COVIDSafe/mobile-android/blob/b827cf3ccef72a3d38c6fc37466a99868823540f/app/src/main/java/au/gov/health/covidsafe/streetpass/StreetPassWorker.kt#L596